lua/autorun/server/sv_cfc_explosion_antispam_init.lua

```diff
@@ -0,0 +1,76 @@
+CFC_ExplosionAntispam = CFC_ExplosionAntispam or {}
+
+CFC_ExplosionAntispam.NEAR_DIST = math.pow( 250, 2 ) --How close explosive damage events have to be for limitations to apply
+CFC_ExplosionAntispam.MAX_NEAR = 10 --The max number of explosive damage events allowed in an area
+CFC_ExplosionAntispam.EXPLOSION_TIMEOUT = 1 --How long explosive damage events stay logged for
+CFC_ExplosionAntispam.BARREL_TIMOUT = 2 --Base timeout duration for explosive barrels
+CFC_ExplosionAntispam.BARREL_TIMOUT_OFFSET_MAX = 1 --Adds random extra timeout duration from 0 to this value to barrels to look more natural
+CFC_ExplosionAntispam.STOP_ALL_DAMAGE_TIMEOUT = 10 --Default time that stopAllDamage() lasts for
+
+local stopAllDamage = false
+local recentDamagePositions = {}
+local uniqueKey = 0
```

Comment:
These would be great to plop on the global:
```suggestion
CFCExplosionAntispam.stopAllDamage = false
CFCExplosionAntispam.recentDamagePositions = {}
CFCExplosionAntispam.uniqueKey = 0
```

Reply:
They're specifically kept local so they can't be interfered with. We only want stopAllDamage to be modified through use of the global setter function which turns it on and then turns it off later, and the other functions are something which we don't want anything else to be able to modify.